Rakefile

```diff
@@ -55,6 +55,12 @@ end
 
 namespace :test do
   RuboCop::RakeTask.new
+
+  begin
+    require 'rspec/core/rake_task'
+    RSpec::Core::RakeTask.new(:spec)
+  rescue LoadError
```

Comment:
When would LoadError occur?  If we are explicitly doing nothing in this case, a comment explaining why could be helpful.

Reply:
yeah, it's what rspec recommends so the rakefile can be used if rspec is only in a --without group we might not have in other places in CI or in production.  it's probably YAGNI 🤷‍♂️